### PR TITLE
LLP-35-Locales

### PR DESCRIPTION
**Pull request by mdainys**

### CODE REVIEW

File: src/components/ContactForm.js [hunk 1]
```diff
@@ -1,35 +1,39 @@
...
-            </p>
+            <h3>{t.inputForm.title}</h3>
+            <p>{t.inputForm.description}</p>
          </div>
          }  
          <form className={styles.formBlock}>
             <div className={styles.formInputBlock}>
             <input type="text" 
-            placeholder="Topic"
+            placeholder={t.inputForm.topic}
```
Comment (pinenetcoder) [inline src/components/ContactForm.js:23]: please call it t.contactForm.topicPlaceholder

File: src/pages/contacts.js [hunk 2]
```diff
@@ -1,49 +1,47 @@
...
+   const t = router.locale === 'lt' ? lt : en
    
    return (
       <IndexLayout>
          <main>
          <section className={styles.contactsPageWrapper}>
             <div className={styles.contactsPageList}>
               <div className={styles.contactsHeroItem}>
-                <h3 className={styles.contactsTitle}>Kontaktai</h3>
+                <h3 className={styles.contactsTitle}>{t.contacts.heroBlock.coantacts}</h3>
```
Comment (pinenetcoder) [inline src/pages/contacts.js:18]: coantacts - typo

File: src/components/ContactForm.js (hunk 1 above)
Comment (mdainys) [reply src/components/ContactForm.js:23]: Done to all inputs Placeholder's

File: src/pages/contacts.js (hunk 2 above)
Comment (mdainys) [reply src/pages/contacts.js]: Done

**Review by pinenetcoder — APPROVED** [review]

**Pull request merged**